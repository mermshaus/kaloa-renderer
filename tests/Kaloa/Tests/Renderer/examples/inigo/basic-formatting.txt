A paragraph.

Another paragraph.

This is
one single paragraph with a <br /> break.

[icode]$x[/icode] and [icode]$y[/icode] are variables.

[b]strong text[/b]

[i]italic text[/i]

[u]underlined[/u]

[s]strike-through[/s]

[strike]strike-through alias[/strike]

[u][s]underline & strike-through[/s][/u]

[b]strong [i]italic[/i] text[/b]

[em]emphasized [strong]strong[/strong] text[/em]

[off][b]Foo & Bar[/b][/off]
