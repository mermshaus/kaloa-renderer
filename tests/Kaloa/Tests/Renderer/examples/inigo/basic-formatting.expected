<p>A paragraph.</p>

<p>Another paragraph.</p>

<p>This is<br />
one single paragraph with a &lt;br /&gt; break.</p>

<p><code>$x</code> and <code>$y</code> are variables.</p>

<p><b>strong text</b></p>

<p><i>italic text</i></p>

<p><u>underlined</u></p>

<p><s>strike-through</s></p>

<p><s>strike-through alias</s></p>

<p><u><s>underline &amp; strike-through</s></u></p>

<p><b>strong <i>italic</i> text</b></p>

<p><em>emphasized <strong>strong</strong> text</em></p>

<p>[b]Foo &amp; Bar[/b]</p>